test/hadoop_test.py

```diff
@@ -158,6 +190,19 @@ def test_unicode_job(self):
         self.assertEqual(c[0], "test\t2\n")
 
 
+class MapreduceLocalTest(unittest.TestCase, MapreduceTestMixin):
```

Comment:
I hate multiple inheritance, but sometimes for unit tests it makes things a lot more convenient